### PR TITLE
Support the updates-testing scenario in `test/run`

### PR DESCRIPTION
**Pull request by jelly**
Slowly we are getting there, sorry I forgot that we have a custom `test/run` per project while I should have known and noted in my PR original Cockpit PR.

### CODE REVIEW

**Comment by jelly** [comment]
Hmm I do see `dnf -y update` but I don't see any updates output https://cockpit-logs.us-east-1.linodeobjects.com/pull-1393-20230905-074549-c169a45a-fedora-38-updates-testing/log

**Comment by martinpitt** [comment]
It feels a little weird to me to do it that way -- When accidentally not dropping --fresh, it would silently succeed. Makefile already handles the pybridge scenario, why not updates-testing as well? That's easier to reproduce locally as a developer as well.

That said, fine to land of course -- so far the running test looks good.

**Comment by jelly** [comment]
> It feels a little weird to me to do it that way -- When accidentally not dropping --fresh, it would silently succeed. Makefile already handles the pybridge scenario, why not updates-testing as well? That's easier to reproduce locally as a developer as well.
> 
> That said, fine to land of course -- so far the running test looks good.

Yeah feels a bit hackish right? I'm happy to improve this, I tried using `VM_CUSTOMIZE_FLAGS but for some reason that seems to clash with the normal `image-customize`. I'm happy to wait till I have better internet so I can debug this better.

**Comment by martinpitt** [comment]
I meant, just do this *entirely* in Makefile, just like it handles /pybridge.

**Comment by jelly** [comment]
> I meant, just do this _entirely_ in Makefile, just like it handles /pybridge.

Sure, but what I am wondering is if this requires us to run `vm-customize` twice or can it be done with one invocation?

**Comment by martinpitt** [comment]
One is enough, you can string arbitrarily many options together. It's a trade-off between being faster, and being able to run parts of your setup with `--no-network`.

File: Makefile [hunk 1]
```diff
@@ -164,6 +166,10 @@ VM_DEPENDS = $(COCKPIT_WHEEL)
 VM_CUSTOMIZE_FLAGS = --install $(COCKPIT_WHEEL)
```
Comment (martinpitt) [inline Makefile:166]: This now should also get `--no-network`

**Review by martinpitt — CHANGES_REQUESTED** [review]
Looks nice! just one detail

**Comment by jelly** [comment]
We never tested centos-8-stream so this is expected to fail

**Comment by martinpitt** [comment]
@jelly please add a second commit to drop the pybridge scenario. Cheers!

**Review by martinpitt — APPROVED** [review]
Cheers!

**Pull request merged**